sphinxcontrib/confluencebuilder/config.py

```diff
@@ -123,4 +123,36 @@ def validate(c, log=True):
 properly set with the publisher's Confluence username.
 """)
 
+            if c.confluence_ca_cert:
+                if (not os.path.isfile(c.confluence_ca_cert) and
+                        not os.path.isdir(c.confluence_ca_cert)):
+                    errState = True
+                    if log:
+                        ConfluenceLogger.error(
+"""missing certificate authority
+
+The option 'confluence_ca_cert' has been provided to find a certificate 
+authority file or path from a relative location. Ensure the value is set to a 
+proper file path.
+""")
+
+            if c.confluence_client_cert:
+                if isinstance(c.confluence_client_cert, tuple):
+                    cert_files = c.confluence_client_cert
+                else:
+                    cert_files = (c.confluence_client_cert,)
+
+                for cert_file in cert_files:
+                    if not os.path.isfile(cert_file):
+                        errState = True
+                        if log:
+                            ConfluenceLogger.error(
```

Comment:
Why not raise in this condition instead of having a special error condition and logic flow? If the error is non-recoverable then you should raise the error

Reply:
I didn’t raise an error for consistencies sake. None of the above error conditions actually raise an error, they just set the `errorState` to `True` which in turn raises an error. I assumed this was what Sphinx wanted.

I’d be happy to change it, but I think I should also change all the other error conditions in this file as well.

Reply:
I think what was submitted is correct (with respect to setting the error flag and logging the error). This is to allow multiple configuration items to be verified (and output at once) before a configuration exception (`ConfluenceConfigurationError`) is thrown (in the parent class).